UI/script.js

```diff
@@ -0,0 +1,354 @@
+class AdsList{
```

Comment:
I'd advice you to create a new file for each JS class, just to make code more readable.